project/test/resources/OPConsultantRecord-Output.json

```diff
@@ -1,2688 +1,2143 @@
+            "display": "Amoxicillin (as amoxicillin trihydrate) 25 mg/mL oral suspension"
+          } ],
+          "text": "Amoxicillin"
+        } ],
+        "site": {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "323732002",
+            "display": "Amoxicillin (as amoxicillin trihydrate) 25 mg/mL oral suspension"
+          } ],
+          "text": "Amoxicillin"
+        },
+        "route": {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "323732002",
+            "display": "Amoxicillin (as amoxicillin trihydrate) 25 mg/mL oral suspension"
+          } ],
+          "text": "Amoxicillin"
+        },
+        "method": {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "323732002",
+            "display": "Amoxicillin (as amoxicillin trihydrate) 25 mg/mL oral suspension"
+          } ],
+          "text": "Amoxicillin"
+        },
+        "doseAndRate": [ {
+          "doseQuantity": {
+            "value": 500.0,
+            "unit": "mg",
+            "system": "http://unitsofmeasure.org",
+            "code": "mg"
+          },
+          "rateQuantity": {
+            "value": 3.0,
+            "unit": "1/d",
+            "system": "http://unitsofmeasure.org",
+            "code": "{1/d}"
+          }
+        } ]
+      } ]
+    }
+  }, {
+    "fullUrl": "urn:uuid:3bef0ee5-0a80-47a3-bdb0-ee36eb6cbe93",
+    "resource": {
+      "resourceType": "MedicationRequest",
+      "id": "3bef0ee5-0a80-47a3-bdb0-ee36eb6cbe93",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2024-07-01T16:29:32.647+05:30",
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/MedicationRequest" ]
+      },
+      "status": "active",
+      "intent": "proposal",
+      "medicationCodeableConcept": {
+        "coding": [ {
+          "system": "http://snomed.info/sct",
+          "code": "785686003",
+          "display": "Amoxicillin anhydrous"
+        } ],
+        "text": "Amoxicillin anhydrous"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "authoredOn": "2024-07-01T16:29:32+05:30",
+      "requester": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "reasonCode": [ {
+        "coding": [ {
+          "system": "http://snomed.info/sct",
+          "code": "785686003",
+          "display": "Amoxicillin anhydrous"
+        } ],
+        "text": "Amoxicillin anhydrous"
+      } ],
+      "reasonReference": [ {
+        "reference": "urn:uuid:b3679f88-6c04-4a76-93e6-d0472340a843",
+        "display": "Condition"
+      } ],
+      "dosageInstruction": [ {
+        "additionalInstruction": [ {
+          "coding": [ {
+            "system": "http://snomed.info/sct",
+            "code": "785686003",
```

Comment:
wrong code for instructions. Ideally you should include from https://hl7.org/fhir/R4/valueset-additional-instruction-codes.html. if not, avoid putting additional instructions or add additionInstruction.text